src/styles/font.scss

```diff
@@ -0,0 +1,3 @@
+$description-font: "Abel", sans-serif;
```

Comment:
This file is redundant, We already have a `fonts` file, You can go ahead and remove it.